week4/my-app/src/components/CityList.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Cities from "./Cities";
+import Form from "./Form";
+
+function CityList() {
+  const [weather, setWeather] = useState([]);
+  const [status, setStatus] = useState("loading");
+  const [errorMessage, setErrorMessage] = useState(''); 
+
+  async function getCity(city) {
+    try {
+      const response = await fetch(
+        `https://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=${process.env.REACT_APP_OPENWEATHERMAP_API_KEY}`
+      );
+      if (response.ok) {
+        const weatherInfo = await response.json();
+        setWeather([weatherInfo, ...weather]);
+        setStatus("success");
+        setErrorMessage('');
+      } else {
+        setErrorMessage('Error fetch data');
+        alert("Please insert valid city name");
+      }
+    } catch (error) {
+      setStatus("error");
+      setErrorMessage(error.message);
+    }
+  }
+
+  function deleteCity(ID) {
+    const newArr = weather.filter(weather => weather.id !== ID);
+    setWeather(newArr);
+  }
+
+  return (
+    <div className="city-item">
+      <h1>Weather</h1>
+      {status === "Loading" && <h1>Loading</h1>}
```

Comment:
this will never be invoked, look up the word case -> `Loading` here ≠ `loading` line 7.
```suggestion
      {status === "loading" && <h1>Loading</h1>}
```

Reply:
done